lte/gateway/c/oai/tasks/sgw/sgw_context_manager.c

```diff
@@ -354,28 +340,26 @@ int sgw_cm_remove_eps_bearer_entry(
 s_plus_p_gw_eps_bearer_context_information_t* sgw_cm_get_spgw_context(
     teid_t teid) {
   s_plus_p_gw_eps_bearer_context_information_t* spgw_bearer_context_info = NULL;
-  hash_table_ts_t* state_imsi_ht = get_spgw_ue_state();
+  hash_table_ts_t* state_teid_ht = get_spgw_teid_state();
```

Comment:
slightly unrelated to this PR but
```
lte/gateway/c/oai/tasks/sgw/sgw_context_manager.c#L326

unused parameter 'sgw_pdn_connection' [-Wunused-parameter]
```
is the lint accurate?

Reply:
yes, it's legit, I will take care of that on a separate PR, thanks for flagging!